src/hooks/useTheming.js

```diff
@@ -0,0 +1,92 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
If the `React` import below is removed, can we get rid of this line?

Reply:
Yup, I removed the import and also the linting exception.